frontend/src/pages/articles/articles-page.tsx

```diff
@@ -0,0 +1,39 @@
+import { Link, RouterOutlet } from '~/libs/components/components.js';
+import { ArticleSubRoute } from '~/libs/enums/enums.js';
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
+import { useLocation } from '~/libs/hooks/hooks.js';
+
+import styles from './styles.module.scss';
+
+const ArticlesPage: React.FC = () => {
+  const { pathname } = useLocation();
+  const activePage = pathname.split('/').pop();
+
+  return (
+    <div className={styles.wrapper}>
+      <div className={styles.tabsWrapper}>
+        <Link
+          to={ArticleSubRoute.FEED}
+          className={getValidClassNames(
+            styles.tab,
+            activePage === ArticleSubRoute.FEED && styles.activeTab,
```

Comment:
I believe `Navlink` as an option injects a callback passing for `className` prop with `isActive` argument you may check if we can accomplish active link styling with this approach
Like
```
pass optional 'activeClassName' style for Link component 
And in Link component change className passing to something like 
({ isActive }) => getValidClassNames(styles.link, className, isActive && activeClassName)
```